src/styles/_progressbar.scss

```diff
@@ -0,0 +1,42 @@
+.main-body {
+    margin-top: 2px;
+}
+
+#progress-bar-container {
+    height: 2px;
+    position: sticky;
+    width: 100%;
+    box-shadow: 0 -1px 18px 0 #000000BD;
+    z-index: 1029;
+    top: 68px;
+}
+
+#progress-bar {
+    height: 2px;
+    width: 0%;
+    background: #008647;
+
+    @media (max-width: $screen-md) {
+        height: 3px;
+    }
+}
+
+body {
+    background: linear-gradient(-155deg, #fff 50%, #008647 50%);
+    background-size: 100% calc(100% - 100vh + 74px);
+    background-repeat: no-repeat;
+
+    @media (max-width: $screen-md) {
+        background: linear-gradient(-140deg, #fff 50%, #008647 50%);
+    }
+}
+
+body::before {
```

Comment:
is there a way to use 'global-nav--component' instead of boxy?

Reply:
they perform to different functions.